### PR TITLE
Update linux build instructions with latest SDK fixed LD_LIBRARY_PATH

### PR DESCRIPTION
**Pull request by t-kopp**

### CODE REVIEW

File: notes/linuxdev/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ category: notes
   - Qt Creator from the [Qt project](https://www.qt.io/download-open-source/) or your distribution packages.
   - [Our special build of the Qt 5.9.7 SDK that includes QtWebKit](https://s3.amazonaws.com/misc.meltymedia/shotcut-build/qt-5.9.7-ubuntu16.04-x86_64.txz)    
     Extract it to $HOME/Qt
-  - [Shotcut SDK (733 MiB! current version 20.09.13)](https://s3.amazonaws.com/builds.us.meltytech/shotcut/shotcut-linux-x86_64-sdk-200913.txz)    
+  - [Shotcut SDK (739 MiB! current version 20.09.27)](https://s3.amazonaws.com/builds.us.meltytech/shotcut/shotcut-linux-x86_64-sdk-200927.txz)
```
Comment (ddennedy) [inline notes/linuxdev/index.md:11]: The trailing spaces are significant and cause a line break.

File: notes/linuxdev/index.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Configure all of your settings in the Build & Run section.
    - Under **Deployment** click **Add Deploy Step &gt; Make**, and in **Make arguments** add "install".  
    - Under **Run &gt; Run configuration** click **Add &gt; Custom Executable &gt; Executable** add   `$HOME/Projects/Shotcut/Shotcut.app/bin/shotcut`
    - Under **Run Environment &gt; Details** add the following environment variables:
-     - Set `LD_LIBRARY_PATH` to `${HOME}/Projects/Shotcut/Shotcut.app/lib:${QTDIR}/lib:${HOME}/Projects/Shotcut/Shotcut.app/src/build-shotcut-Desktop_Qt_5_6_1_GCC_64bit-Debug/CuteLogger`
+     - Set `LD_LIBRARY_PATH` to `${HOME}/Projects/Shotcut/Shotcut.app/lib:${QTDIR}/lib:${HOME}/Projects/Shotcut/Shotcut.app/src/build-shotcut-Desktop_QT-Debug/CuteLogger`
```
Comment (ddennedy) [inline notes/linuxdev/index.md:31]: This path change is not correct for the way my Qt Creator creates per the Kit name by default based on the Qt version. Mine is `build-shotcut-Desktop_Qt_5_9_7_GCC_64bit-Debug`.

**Review by ddennedy — CHANGES_REQUESTED** [review]
The Qt version is changing in the next version, and we are no longer supplying a custom Qt SDK. You can use the one in a recent Linux distro or from Qt installer/MaintenanceTool. For these reasons and a couple of others, I plan to discontinue the Linux SDK and remove this page in a couple of days. If you are going to start hacking on Shotcut, you should go in that direction. You will need the very latest release of MLT, however - preferably git master branch. You can use our build script in `scripts/` as well.

File: notes/linuxdev/index.md (hunk 2 above)
Comment (t-kopp) [reply notes/linuxdev/index.md:31]: Whoops - my bad, sorry for the noise. I had indeed truncated it during import into QTcreator

**Comment by t-kopp** [comment]
Thanks for the explanations. I'll try to get the latest Git Masterbranch to compile with the newest QT on my system. I already used the latest master branch with the 0927 sdk which compiled fine. But I'm fine with changing the QT version.

**Pull request closed without merging**